src/components/PlantsV2/components/HighestAndLowestMortalityRateCard.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import strings from 'src/strings';
+import { Box, Divider, Typography, useTheme } from '@mui/material';
+import { ObservationResults } from 'src/types/Observations';
+import { useAppSelector } from 'src/redux/store';
+import { selectObservationPlantingZone } from 'src/redux/features/observations/observationPlantingZoneSelectors';
+import { useDefaultTimeZone } from 'src/utils/useTimeZoneUtils';
+
+type HighestAndLowestMortalityRateCardProps = {
+  observation?: ObservationResults;
+};
+
+export default function TotalMortalityRateCard({ observation }: HighestAndLowestMortalityRateCardProps): JSX.Element {
+  const theme = useTheme();
+  const defaultTimeZone = useDefaultTimeZone();
+
+  let highestMortalityRate = 0;
+  let highestZoneId: number;
+  observation?.plantingZones.forEach((zone) => {
+    if (zone.mortalityRate > highestMortalityRate) {
+      highestMortalityRate = zone.mortalityRate;
+      highestZoneId = zone.plantingZoneId;
+    }
+  });
+
+  let lowestMortalityRate = 100;
+  let lowestZoneId: number;
+  observation?.plantingZones.forEach((zone) => {
+    if (zone.mortalityRate < lowestMortalityRate) {
+      lowestMortalityRate = zone.mortalityRate;
+      lowestZoneId = zone.plantingZoneId;
+    }
+  });
```

Comment:
Likewise, if all zones had mortality rate `100`, then none would get picked.

Reply:
If no zone is picked, nothing shows because the render has this validation` lowestPlantingZone && `